src/map/elemental.hpp

```diff
@@ -42,22 +43,19 @@ enum elemental_elementalid  : uint16 {
 };
 
 struct elemental_skill {
-	unsigned short id, lv;
-	short mode;
+	uint16 id, lv;
 };
 
 struct s_elemental_db {
-	int class_;
+	int32 class_;
 	char sprite[NAME_LENGTH], name[NAME_LENGTH];
```

Comment:
I missed this, but should we convert these to std::string? Might require quite a few changes as well, but could be worth while to just do it now.

If we do go this route, don't forget to add name size warning checks during the parsing of these!